sdk/messaging/azservicebus/client_test.go

```diff
@@ -58,6 +59,21 @@ func TestNewClientWithAzureIdentity(t *testing.T) {
 	require.NoError(t, err)
 
 	err = sender.SendMessage(context.TODO(), &Message{Body: []byte("hello - authenticating with a TokenCredential")}, nil)
+
+	if err != nil {
+		if strings.Contains(err.Error(), "'Send' claim(s) are required to perform this operation") {
```

Comment:
Nit, you could combine this into one `if` clause to reduce indentation.

Reply:
Done.